early-init.el

```diff
@@ -62,10 +62,21 @@
 (unless (file-exists-p rational-config-path)
   (mkdir rational-config-path t))
 
-(defvar rational-prefer-guix-packages nil
+(defun rational-runs-guix-emacs-p ()
```

Comment:
Maybe `rational-using-guix-emacs-p` is slightly more clear?

Reply:
OK. Should I also change `file-chase-links` to `file-truename`? In the documentation I understand that the latter does a better job with symlinks in the directories of the path.

Reply:
Yep, that is probably the better choice as long as it doesn't prompt the user to follow the symlink.  My guess is that it doesn't, you'll find out right away as soon as you test it :)

Reply:
I will...